### DISCUSSION TITLE
Expanding Calendar Grid

### DISCUSSION THREAD

**Discussion by monroee** [nhn/tui.calendar#1001] [Q&A]
Hello, is there a configuration or a way to expand the calendar grid for the daily and weekly view? 

I need to put more information on the schedule title element. 

Thanks in advance! :)

**Comment by adhrinae** [comment]
How exactly would you like to expand the grid?

If you want to customize the title, the `template` option might be helpful for you.

https://nhn.github.io/tui.calendar/latest/Template - Check `schedule` property
https://nhn.github.io/tui.calendar/latest/tutorial-example15-template-for-timegrid - Example

**Reply by monroee** [reply]
> How exactly would you like to expand the grid?
> 
> If you want to customize the title, the `template` option might be helpful for you.
> 
> https://nhn.github.io/tui.calendar/latest/Template - Check `schedule` property https://nhn.github.io/tui.calendar/latest/tutorial-example15-template-for-timegrid - Example

I wanted to resize the time grid's height and width,to make it wider because I will be making the schedule titles fixed height and width to 100px.
for the schedule, I already have resized it by "customStyle" attribute, but the grid didn't adjust.

**Reply by adhrinae** [reply]
I see. I'm afraid of saying that it's not adjustable.
It's depending on the container element's width, and calculate and divide the width of columns equally(except the `narrowWeekend` option).

**Reply by monroee** [reply]
> I see. I'm afraid of saying that it's not adjustable. It's depending on the container element's width, and calculate and divide the width of columns equally(except the `narrowWeekend` option).

Alright. Thank you for your time!
